src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -112,6 +112,16 @@ class HuiGenericEntityRow extends LitElement {
                     : this.hass.localize(
                         "ui.panel.lovelace.cards.entities.never_triggered"
                       )
+                  : this.config.secondary_info === "position" &&
+                    stateObj.attributes.current_position
+                  ? this.hass.localize("ui.card.cover.position") +
+                    ": " +
+                    stateObj.attributes.current_position
+                  : this.config.secondary_info === "tilt-position" &&
+                    stateObj.attributes.current_tilt_position
+                  ? this.hass.localize("ui.card.cover.tilt_position") +
+                    ": " +
+                    stateObj.attributes.current_tilt_position
```

Comment:
```suggestion
                  ? `${this.hass.localize("ui.card.cover.tilt_position")}: ${stateObj.attributes.current_tilt_position}
```

Reply:
The expressions are already escaped starting with L94
https://github.com/home-assistant/frontend/blob/f8b148ad76ee5a1f8c441077f61ca6535c3c089e/src/panels/lovelace/components/hui-generic-entity-row.ts#L94-L125
Am I missing something?

Reply:
You should not use string concatenation, it is just a different way of writing the same.